pkgs/development/libraries/science/math/or-tools/default.nix

```diff
@@ -1,109 +1,95 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, cmake
-, abseil-cpp
+{ abseil-cpp
 , bzip2
-, zlib
-, lsb-release
-, which
-, protobuf
 , cbc
+, cmake
+, eigen
 , ensureNewerSourcesForZipFilesHook
+, fetchFromGitHub
+, fetchpatch
+, glpk
+, lib
+, pkg-config
+, protobuf
 , python
+, re2
+, stdenv
 , swig4
+, unzip
+, zlib
 }:
 
 stdenv.mkDerivation rec {
   pname = "or-tools";
-  version = "9.1";
-  disabled = python.pythonOlder "3.6";  # not supported upstream
+  version = "9.4";
+  disabled = python.pythonOlder "3.6"; # not supported upstream
```

Comment:
This can be removed. Python3 is already 3.10.

Reply:
This derivation is aliased as `python2Packages.ortools`.

https://github.com/NixOS/nixpkgs/blob/fd54651f5ffb4a36e8463e0c327a78442b26cbe7/pkgs/top-level/python-packages.nix#L6515

Reply:
You are sure? python-package is for python3 only and python2Packages is mostly broken except some specific packages.